h2o-r/h2o-package/R/word2vec.R

```diff
@@ -25,13 +25,13 @@
 #' h2o.init()
 #' 
 #' # Import the CraigslistJobTitles dataset
-#' job.titles.path = "https://raw.githubusercontent.com/h2oai/sparkling-water/rel-1.6/examples/smalldata/craigslistJobTitles.csv"
-#' job.titles <- h2o.importFile(job.titles.path, destination_frame = "jobtitles",
+#' f <- "https://raw.githubusercontent.com/h2oai/sparkling-water/rel-1.6/examples/smalldata/"
+#' jobtitles <- h2o.importFile(pasteo(f, "craigslistJobTitles.csv"), 
```

Comment:
should this be paste0 instead of pasteo?

Reply:
@angela0xdata Yep, oops I thought I already changed that.  Maybe I updated that code in two places.  Thx.